.travis.yml

```diff
@@ -1,60 +1,53 @@
 sudo: false
 language: python
-
 cache:
   directories:
-    - .benchmarks
+  - ".benchmarks"
 branches:
   only:
-    - master
-
+  - master
 before_install:
-  - pip install --upgrade pip setuptools wheel
-
+- pip install --upgrade pip setuptools wheel
 matrix:
   include:
   - os: linux
     env: NAME=mypy
-    python: "3.6"
+    python: '3.6'
     install:
-      - cat dev_tools/conf/pip-list-dev-tools.txt | grep mypy | xargs pip install
+    - cat dev_tools/conf/pip-list-dev-tools.txt | grep mypy | xargs pip install
     script: check/mypy
-
   - os: linux
     env: NAME=pylint
-    python: "3.6"
+    python: '3.6'
     install:
-      - cat dev_tools/conf/pip-list-dev-tools.txt | grep "pylint\|astroid" | grep -v "#" | xargs pip install
+    - cat dev_tools/conf/pip-list-dev-tools.txt | grep "pylint\|astroid" | grep -v
+      "#" | xargs pip install
     script: check/pylint
-
   - os: linux
     env: NAME=format
-    python: "3.6"
+    python: '3.6'
     install:
-      - cat dev_tools/conf/pip-list-dev-tools.txt | grep yapf | xargs pip install
+    - cat dev_tools/conf/pip-list-dev-tools.txt | grep yapf | xargs pip install
     script: check/format-incremental
-
   - os: linux
     env: NAME=pytest-and-incremental-coverage
-    python: "3.6"
+    python: '3.6'
     install:
-      - pip install -r requirements.txt
-      - pip install -r cirq/contrib/contrib-requirements.txt
-      - pip install -r dev_tools/conf/pip-list-dev-tools.txt
+    - pip install -r requirements.txt
+    - pip install -r cirq/contrib/contrib-requirements.txt
+    - pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/pytest-and-incremental-coverage master --actually-quiet
-
   - os: linux
     env: NAME=pytest (without contrib)
-    python: "3.7"
+    python: '3.7'
     install:
-      - pip install -r requirements.txt
-      - pip install -r dev_tools/conf/pip-list-dev-tools.txt
+    - pip install -r requirements.txt
+    - pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/pytest --ignore=cirq/contrib --benchmark-skip --actually-quiet
-
   - os: windows
     env: NAME=pytest (Windows)
     language: sh
-    python: "3.7.3"
+    python: 3.7.3
```

Comment:
This one didn't get quotes probably because it doesn't look like a number.  Could this be an issue if someone updated the version to `python: 3.8`?